config/v1/types_build.go

```diff
@@ -72,6 +76,27 @@ type ImageLabel struct {
 	Value string `json:"value,omitempty"`
 }
 
+type RegistriesList []string
```

Comment:
Also, are you sure you want this?  No reason you can't have it, but it may mean a bunch of casts.

Reply:
you mean vs just using a string type directly?  I assume this was following a pattern we've established elsewhere, i don't have a preference.

Reply:
> you mean vs just using a string type directly? I assume this was following a pattern we've established elsewhere, i don't have a preference.

We've done it both ways and unless we're enumerating valid values, I've found `[]string` to be easier to deal with.